CBLClient/Apps/CBLTestServer-Dotnet/TestServer/Router.cs

```diff
@@ -339,6 +342,55 @@ internal static void Handle([NotNull]Uri endpoint, [NotNull]Stream body, [NotNul
             return;
         }
 
+        private static void CopyFiles([NotNull]NameValueCollection args,
+            [NotNull]IReadOnlyDictionary<string, object> postBody,
+            [NotNull]HttpListenerResponse response)
+        {
+            string sourcePath = postBody["source_path"].ToString();
+            string destinationPath = postBody["destination_path"].ToString();
+            DirectoryCopy(sourcePath, destinationPath, true);
+            response.WriteBody("Copied");
```

Comment:
same as above. if there is a possible of checking the status, that will be good to get the right status

Reply:
it raises exception, no need to check copy status